common/metrics/config.go

```diff
@@ -399,3 +443,64 @@ func configExcludeTags(cfg ClientConfig) map[string]map[string]struct{} {
 	}
 	return tagsToFilter
 }
+
+func (s SanitizeRange) toTally() (tally.SanitizeRange, error) {
+	startRangeRunes := []rune(s.StartRange)
+	if len(startRangeRunes) != 1 {
+		return tally.SanitizeRange{}, fmt.Errorf("start range '%+v' must be a single rune", startRangeRunes)
```

Comment:
does this give enough feedback for someone using something that looks like a single character but isn't ala https://en.wikipedia.org/wiki/Combining_character - maybe it does but just want to bring it up.

Reply:
i think the definition of a rune should make that clear as it maps to a Unicode Code Point